offline/docs.md

```diff
@@ -576,3 +576,31 @@ d helm upgrade --install sftd ./charts/sftd \
   --set-file tls.key=/path/to/tls.key \
   --values values/sftd/values.yaml
 ```
+
+### (Optional) Health-Check deployed k8s cluster
+There might be possibility that the provided compute resources does not match the requried resources for deploying whole application, that might lead to cluster being in unhealthy state. Some of the healh checks are -
```

Comment:
```suggestion
It could be that the provided Kubernetes cluster resources do not match the required resources of the application. This may lead to an unhealthy state. Some of the health checks are:
```